internal/js_scanner/js_scanner_test.go

```diff
@@ -552,6 +566,24 @@ export const foo = 0
 	}
 }
 
+func FuzzHoistExport(f *testing.F) {
+	tests := fixturesHoistExport()
+	for _, tt := range tests {
+		f.Add(tt.source) // Use f.Add to provide a seed corpus
+	}
+	f.Fuzz(func(t *testing.T, source string) {
+		result := HoistExports([]byte(source))
+		got := []byte{}
+		for _, imp := range result.Hoisted {
+			got = append(got, bytes.TrimSpace(imp)...)
+			got = append(got, '\n')
+		}
+		if utf8.ValidString(source) && !utf8.ValidString(string(got)) {
```

Comment:
not sure what additional properties I can assert on here and in `FuzzHoistExport`. Currently I'm basically just testing that no errors are thrown plus that we get unicode out